### PR TITLE
docs: if, set and system stores

### LINKED ISSUE #366: docs: if, set, and Using variable stores topics do not document use of system stores
* They need to document use of `&layer` etc.
* Some of the 'related topics' say 'Variable system stores' but should say 'Variable stores'
* They need to note that `if()` must be at start of rule (see also https://help.keyman.com/developer/language/guide/rules):

```
Certain statements must appear in specific positions in the rule. The following order is enforced by the compiler:

* nul
* if(), baselayout() and platform()
* All other statements and characters
```

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #366.

Note: I have not attempted to add any additional content to the guide/variable-stores topic, as it doesn't really make sense to document the system store modes for `if()` and `set()` there.

### CODE REVIEW

File: developer/language/reference/if.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+```
+if(&platform = 'web') + any(letterGroup1) > index(outputGroup1, 2)
+```
+
+## Platforms
+
+The `if()` statement can be used in keyboards on the following platforms.
+
+Currently, only Windows supports the options.htm form to set variable stores.
```
Comment (darcywong00) [inline developer/language/reference/if.md:99]: Keyman for Linux supports options dialog in keymanapp/keyman#2566

Are variable stores stored differently?

Comment (mcdurdin) [reply developer/language/reference/if.md]: > Are variable stores stored differently?

I'm not sure I understand the question?

Comment (darcywong00) [reply developer/language/reference/if.md]: I know on Linux the options.htm form can toggle and save keyboard options.

I was checking if variable stores are stored the same way.

Comment (mcdurdin) [reply developer/language/reference/if.md]: variable stores === keyboard options :)

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**